apps/system/style/notifications/notifications.css

```diff
@@ -146,15 +147,20 @@
 
 #notification-bar button {
   flex: initial;
-  width: auto;
+  width: 9.5rem;
```

Comment:
we need to keep the `width:auto` because in some languages "Clear all" might be too wide otherwise.